packages/packagers/css/src/CSSPackager.js

```diff
@@ -140,8 +140,7 @@ async function processCSSModule(
   let defaultImport = null;
   if (usedSymbols.has('default')) {
     let incoming = bundleGraph.getIncomingDependencies(asset);
-    // `import * as ns from ""; ns.default` is fine.
-    defaultImport = incoming.find(d => d.meta.hasDefaultImport);
+    defaultImport = incoming.find(d => d.symbols.hasExportSymbol('default'));
```

Comment:
The reason `hasDefaultImport` was added originally:
```css
.a {
	color: red;
}

.default {
	background: blue;
}
```

```js
import * as css from "./index.module.css";

console.log(css.default);
```

```
@parcel/packager-css: CSS modules cannot be tree shaken when imported with a default specifier
index.js:3:13
  2 |
> 3 | console.log(css.default);
>   |             ^^^^^^^^^^^
  4 |
Instead do: import * as style from "./index.module.css";
```

Reply:
It seemed weird to treat them differently TBH.

```js
import x from 'foo';
console.log(x);
```

and 

```js
import * as ns from 'foo';
console.log(ns.default);
```

should behave the same. But this is one of those interop issues we have in general at the moment. Mainly I didn't feel like wiring up all of this tracking in rust and passing it through to JS and everything for such a minor edge case.

Reply:
Should we change the CSS module JS file to be ESM instead before stable? (so forcing you to use `import * as ns`)

Reply:
Maybe? Might break a lot of code though